source/utilities.tex

```diff
@@ -1730,13 +1730,13 @@
 \begin{codeblock}
 template<class F, class Tuple, size_t... I>
 constexpr decltype(auto)
-    apply_impl(F&& f, Tuple&& t, index_sequence<I...>) {                // exposition only
+    @\placeholdernc{apply-impl}@(F&& f, Tuple&& t, index_sequence<I...>) {                // \expos
   return @\placeholdernc{INVOKE}@(std::forward<F>(f), std::get<I>(std::forward<Tuple>(t))...);  // see \ref{func.require}
```

Comment:
I find the indentation here visually confusing. That's pre-existing, but changing to italics seems to exacerbate it.